test/helpers/elasticsearch.go

```diff
@@ -21,9 +21,9 @@ import (
 )
 
 const (
-	InfraIndexPrefix          = ".operations."
-	ProjectIndexPrefix        = "project."
-	AuditIndexPrefix          = ".audit."
+	InfraIndexPrefix          = "infra-"
```

Comment:
@jcantrill Are we going to need to update index patterns here?

https://github.com/openshift/origin-aggregated-logging/blob/feature-es6x/elasticsearch/index_templates/com.redhat.viaq-openshift-operations.template.json#L5

Reply:
We actually neeed to make this a dash for audit https://github.com/openshift/origin-aggregated-logging/blob/feature-es6x/elasticsearch/index_templates/com.redhat.viaq-openshift-operations.template.json#L7

Reply:
@jcantrill Just for the `audit.infra-*` (-> `audit-infra-*`) case or other index patterns as well?